### DISCUSSION TITLE
Show local ip for openvpn clients

### DISCUSSION THREAD

**Discussion by Shizanit** [AdguardTeam/AdGuardHome#4909] [Q&A] [answered]
Is it possible to display the local ip of openvpn clients
*adgurd in docker, openvpn without
![image](https://user-images.githubusercontent.com/65107731/189528451-eb207dd0-1ef0-41f7-a0d6-1be246e1228e.png)

**Comment by ainar-g** [comment, marked as answer]
Good day.  What do you mean by “local IP”?

(Можем перейти на русский, если вам так удобнее.)

**Reply by Shizanit** [reply]
I mean local ip on the vpn(tun) network

**Reply by ainar-g** [reply]
I'm still not sure I follow, sorry.  Are you running your container with `--network=host`?  Because otherwise AGH will see all queries as coming from one IP.

**Reply by Shizanit** [reply]
> I'm still not sure I follow, sorry. Are you running your container with `--network=host`? Because otherwise AGH will see all queries as coming from one IP.

If it works, I'll try it later and report back

**Reply by Shizanit** [reply]
All work!
